### ISSUE TITLE
Caminfo does not report camera statistics in isis7.x.x

### ISSUE THREAD

**Issue by lwellerastro** [DOI-USGS/ISIS3#4919]
**ISIS version(s) affected**: isis7.0.0-RC1, isis7.0.0-RC2

**Description**  
Caminfo no longer reports information for the camera when camstats=true

**How to reproduce**  
See data under my user work area /Isis3Tests/Caminfo/CamstatsOutput/

```
caminfo from=TC2W2B0_01_07462S654E3340.lev0.cub to=info_7.0.0RC1_cstats.pvl geometry=false camstats=true linc=100 sinc=100

cat info_7.0.0RC2_cstats.pvl
Object = Caminfo
  Object = Parameters
    Program     = caminfo
    IsisVersion = "7.0.0  | 2022-04-16"
    RunDate     = 2022-04-19T20:41:52
    IsisId      = KAGUYA/TC2/2009-06-04T23:21:14.439997
    From        = TC2W2B0_01_07462S654E3340.lev0.cub
    Lines       = 4656
    Samples     = 4096
    Bands       = 1
  End_Object
End_Object
End
```
Other versions of output are available for the same image: info_5.0.2_cstats.pvl  info_6.0.0_cstats.pvl  info_7.0.0RC1_cstats.pvl  info_7.0.0RC2_cstats.pvl

isis5.0.2 and isis6.0.0 work as expected.

**Additional context**  
It appears the new parameter usecamstatstbl is creating the problem.  
These produce output (and more keywords than I'm expecting):
```
caminfo from=TC2W2B0_01_07462S654E3340.lev0.cub to=info_7.0.0RC1_uselabel_cstats.pvl geometry=false camstats=true usecamstatstbl=true
caminfo from=TC2W2B0_01_07462S654E3340.lev0.cub to=info_7.0.0RC2_uselabel_cstats.pvl geometry=false camstats=true usecamstatstbl=true
```
The above output is in the same data directory mentioned above.

**Comment by lwellerastro** [comment]
bumping this in case there is a desire to get this newly introduced bug fixed before the next release.

**Comment by jessemapel** [comment]
This isn't going to get fixed for the 7.0 release. We'll likely get it fixed before 7.1, though.
